tests/cover/test_numerics.py

```diff
@@ -19,9 +19,12 @@
 
 import math
 
+import pytest
+
 from hypothesis import given, assume
-from tests.common.utils import fails, fails_with
-from hypothesis.strategies import decimals, fractions, float_to_decimal
+from tests.common.utils import fails
+from hypothesis.strategies import decimals, fractions
+from hypothesis.internal.compat import float_to_decimal
 
 
```

Comment:
Yep, works just fine... though you'd better specify the bounds as strings, and set allow_nan=False too!